### ISSUE TITLE
Add symfony3 as requirement in composer.json

### ISSUE THREAD

**Issue by TiMESPLiNTER** [Youshido/GraphQLBundle#7]
The method `FrameworkController::json()` this bundle relies on is available since Symfony 3.0 so the `requires` block in `composer.json` of this bundle should require that symfony >= 3.0 get installed. Else the whole GraphQLBundle will not work.

```
Attempted to call an undefined method named \"json\" of class \"Youshido\\GraphQLBundle\\Controller\\GraphQLController\".
```

The other solution would be to create a symfony 2.7/2.8 comapitble branch or something like that. Or are there any other symfony 3.0 functionalities required to get this bundle working?

**EDIT** no not even symfony 3.0 has this `json` method in controller. Symfony >= 3.1 is required.

**Comment by portey** [comment]
Hi, @TiMESPLiNTER 

Yes, you're right. We'll fix it today.

**Comment by portey** [comment]
Hi, @TiMESPLiNTER 

Thank you for your interesting this bundle. And you're right, we'll fix it today.

**Comment by viniychuk** [comment]
fixed in v1.1.2+
